driver.go

```diff
@@ -10,7 +10,7 @@ type Driver interface {
 
 	// RenderedTextSize returns the size required to render the given string of specified
 	// font size and style.
-	RenderedTextSize(string, float32, TextStyle) Size
+	RenderedTextSize(string, float32, TextStyle) (Size, float32)
```

Comment:
I wonder if we should name the input and output parameters here, as it is confusing what kind of data it needs to receive and what kind of data it will return. Also, I think the comment should be updated too.